features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx

```diff
@@ -543,12 +538,35 @@ class InfoBar extends React.Component {
      */
     toggleOverview = (todo) => {
         if (typeof todo === 'boolean') {
-            this.setState(state => ({ showOverview: todo }));
+            this.setState({ showOverview: todo });
         } else {
             this.setState(state => ({ showOverview: !state.showOverview }));
         }
     };
 
+    /**
+     * [Temporary function] to get the first hybrid https or http endpoint of an API
+     *
+     * @param {Api} api API object
+     * @returns {string}
+     */
+    getHttpsEP = (api) => {
+        const epUrls = api.endpointURLs.filter((v) => {
+            if (v.environmentType === 'hybrid') {
+                return v;
+            } else {
+                return null;
+            }
+        });
+
+        const selectedEnvUrls = epUrls[0].environmentURLs;
```

Comment:
In this case, we could use 

const selectedEnvUrls = api.endpointURLs.find(url => environmentType === 'hybrid')

selectedEnvUrls.https || selectedEnvUrls.http

Reply:
Fixed!